gio/src/auto/list_store.rs

```diff
@@ -63,6 +63,11 @@ impl ListStore {
         }
     }
 
+    //#[doc(alias = "g_list_store_find_with_equal_func_full")]
```

Comment:
It seems like you didn't run `git submodule update -f -i --recursive` before running `generator.py` and used outdated submodules